opentracing/src/main/java/io/opentracing/noop/NoopSpanBuilder.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.noop;
+
+
+import io.opentracing.Span;
+import io.opentracing.Tracer;
+
+public class NoopSpanBuilder implements Tracer.SpanBuilder {
+    private Tracer.SpanBuilder defaultSpanBuilder = new NoopSpanBuilder();
+    private Span defaultSpan = new NoopSpan();
```

Comment:
It is odd that there's two NoopSpan instances created in the JVM. 

A method on the NoopSpan will return a different instance to that returned from the builder.
I'd rather see NoopSpan.DEFAULT_SPAN make package-protected, and it used in the builder here.